compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -177,6 +177,28 @@ class ResolverImpl(
             }
     }
 
+    override fun getFunctionDeclarationsByName(name: KSName, includeTopLevel: Boolean): Sequence<KSFunctionDeclaration> {
+        val qualifier = name.getQualifier()
+        val functionName = name.getShortName()
+        val nonTopLevelResult = this.getClassDeclarationByName(qualifier)?.getAllFunctions()?.filter{ it.simpleName.asString() == functionName }?.asSequence() ?: emptySequence()
```

Comment:
May I know why `getAllFucntions()` is used? Since this API is used to get *declarations*, shouldn't we include only *declared* functions?

Reply:
it is used to get declarations, but I think declarations does not necessary mean comes from source? Like `getClassDeclaration` can get you declarations from classpath.